simulator/risc_v/t/unit_test.cpp

```diff
@@ -396,6 +398,13 @@ TEST_RV64_RR_OP(  8, bclr, 0x0000000000000000, 0x0040000000000000, 0x00000000000
 TEST_RV64_RR_OP(  9, bclr, 0x7fffffffffffffff, 0xffffffffffffffff, 0xffffffffffffffff) // overflow test
 TEST_RV64_RR_OP( 10, bclr, 0x0000000000000000, 0x8000000000000000, 0xffffffffffffffff) // overflow test
 
+TEST_RV32_RR_OP( 1, bset, 0x2237EA14, 0x22376A14, 0xF)
+TEST_RV32_RR_OP( 2, bset, 0x13853FAD, 0x12853FAD, 0x18)
+TEST_RV32_RR_OP( 3, bset, 0xBBC6127, 0xABC6127, 0xAFC18) // index slicing
+
+TEST_RV64_RR_OP( 1, bset, 0x2754179DD4FF1CDD, 0x2554179DD4FF1CDD, 0x39)
+TEST_RV64_RR_OP( 2, bset, 0x15071FE59, 0x15070FE59, 0x10)
```

Comment:
These tests are very unobvious. They give almost no hint on instruction purpose; if they failed, it would be impossible to check what went wrong. Could you please use more transparent values?